search/searcher/search_fuzzy.go

```diff
@@ -67,13 +67,14 @@ func NewFuzzySearcher(ctx context.Context, indexReader index.IndexReader, term s
 		}
 	}
 
-	return NewMultiTermSearcher(ctx, indexReader, candidates, field,
-		boost, options, true)
+	return NewMultiTermSearcherBoosted(ctx, indexReader, candidates, field,
+		boost, fuzzyCandidates.editDistances, options, true)
```

Comment:
line 54 here indicates that `fuzzyCandidates` can be nil... so i think in line 57.5 you can add editDistances = fuzzyCandidates.editDistances there, just like candidates = fuzzyCandidates.candidates